content/guides/ai-studio/get-agents.md

```diff
@@ -0,0 +1,59 @@
+---
+rank: 3
+related_guides:
+  - authentication/tokens/developer-tokens/
+  - box-ai/ai-tutorials/ask-questions
+  - box-ai/ai-tutorials/default-agent-overrides
+  - box-ai/ai-tutorials/generate-text
+  - box-ai/ai-tutorials/extract-metadata-structured
+  - box-ai/ai-tutorials/extract-metadata
+---
+
+# Get agents
+
```

Comment:
Add legal info Box AI Studio is available only for Enterprise Advanced accounts.